templates/scripts/backup.sh

```diff
@@ -19,4 +19,5 @@ for DB in "${DATABASES[@]}"; do
       -U {{ postgres_ha_admin_user }} \
       "${DB}"
 done
+chown dockremap:dockremap "${BKP_DIR}"/*
```

Comment:
We could also add a `--user=postgres` to the `docker exec` call and then grant GID 70 membership for `resetic`.

Reply:
Hm, but that would require a GID 70 to exist, so your solution is shorter.

Reply:
I think we can agree on expecting files be owned by dockremap, even without user remap. This should work in most of the cases.